cl/corpus_importer/management/commands/import_patent.py

```diff
@@ -0,0 +1,133 @@
+import os
+
+from celery import chain
+from django.conf import settings
+from juriscraper.pacer import PacerSession
+
+from cl.corpus_importer.tasks import (
+    filter_docket_by_tags,
+    get_docket_by_pacer_case_id,
+    get_pacer_case_id_and_title,
+    make_fjc_idb_lookup_params,
+)
+from cl.lib.celery_utils import CeleryThrottle
+from cl.lib.command_utils import VerboseCommand, logger
+from cl.recap.constants import PATENT, PATENT_ANDA
+from cl.recap.models import FjcIntegratedDatabase
+from cl.search.models import Docket
+from cl.search.tasks import add_or_update_recap_docket
+
+PACER_USERNAME = os.environ.get("PACER_USERNAME", settings.PACER_USERNAME)
+PACER_PASSWORD = os.environ.get("PACER_PASSWORD", settings.PACER_PASSWORD)
+PATENT_TAGS = ["wdtx_ded_patent"]
+
+
+def get_dockets(options: dict) -> None:
+    """Get Patent litigation from the past 10 years.
+
+    :param options: Options for scraping
+    :return:
+    """
+
+    start = options["skip_until"]
+    stop = options["limit"]
+    """Get the dockets by the particular judge."""
+    q = options["queue"]
+    throttle = CeleryThrottle(queue_name=q)
+    session = PacerSession(username=PACER_USERNAME, password=PACER_PASSWORD)
+    session.login()
+
+    NOS_CODES = [PATENT, PATENT_ANDA]
+    START_DATE = "2012-01-01"
+    items = FjcIntegratedDatabase.objects.filter(
+        nature_of_suit__in=NOS_CODES,
+        date_filed__gt=START_DATE,
+    ).order_by("date_filed")
+    i = 0
+    for item in items:
+        logger.info(f"{i}: Doing FJC_ID with pk: {item.id}")
+        i += 1
+        if i and i < start:
+            # Start processing case at # if not 0.
+            continue
+        if i and i > stop:
+            # Stop processing case at # if not 0
+            continue
+
+        dockets = Docket.objects.filter(idb_data=item)
+        docket_in_system = dockets.exists()
+        if not docket_in_system:
+            params = make_fjc_idb_lookup_params(item)
+            chain(
+                get_pacer_case_id_and_title.s(
+                    pass_through=None,
+                    docket_number=item.docket_number,
+                    court_id=item.district_id,
+                    cookies=session.cookies,
+                    **params,
+                ).set(queue=q),
+                filter_docket_by_tags.s(PATENT_TAGS, item.district_id).set(
+                    queue=q
+                ),
```

Comment:
We can probably omit this one. It just saves us from getting something we got before, but this is a one-off, so I don't think it matters.

Reply:
√ great. removing.